packages/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import type { CommonProps } from '@contentful/f36-core';
+
+import { getTabsStyles } from './Tabs.styles';
+export interface TabsProps extends CommonProps {
+  variant?: 'default' | 'horizontal' | 'vertical';
+  role?: 'navigation' | 'tablist';
+  children?: React.ReactNode;
+}
+
+export function Tabs({
+  className,
+  children,
+  variant = 'default',
+  testId = 'cf-ui-tabs',
+  role = 'tablist',
+  style,
+}: TabsProps): React.ReactElement {
+  const styles = getTabsStyles({ className, variant });
+  console.log('variant', variant);
+  console.log('styles', styles);
+
+  const elementProps = {
+    'data-test-id': testId,
+    className: styles.tabs,
+    style,
+  };
+
+  if (role === 'navigation') {
```

Comment:
If we are to implement some kind of hook to allow automatic selection of tabs, opposed to having a consumer handle it, it would add even more complexity.

Reply:
I'm pretty sure that we need to implement this in v4, but we can do this in the follow up PR.